receiver/otlpreceiver/README.md

```diff
@@ -31,6 +31,11 @@ The following settings are configurable:
 - `endpoint` (default = 0.0.0.0:4317 for grpc protocol, 0.0.0.0:4318 http protocol):
   host:port to which the receiver is going to receive data. The valid syntax is
   described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
+- `path_prefix` (default = "" and only applicable to http protocol):
```

Comment:
what is the behavior if grpc is used? Is it ignored?

Reply:
Should it be `http/path_prefix` ?

Reply:
OK, I would move this down after CORS and provide an example.

Reply:
> what is the behavior if grpc is used? Is it ignored?

Yes.  I'll update the README to be more explicit.

Reply:
> Should it be `http/path_prefix` ?

It is already under the http settings.  Let me try and make that clearer.

Reply:
> OK, I would move this down after CORS and provide an example.

I thought under the HTTP/JSON section might be better given what's there.  If this still doesn't pass muster, I can move to a new section under CORS.

Reply:
@atoulme are you in agreement with the changes that I made above?